srcpkgs/git-remote-hg/template

```diff
@@ -0,0 +1,19 @@
+# Template file for 'git-remote'
+pkgname=git-remote-hg
+version=1.0.2.1
+revision=1
+build_style=gnu-makefile
+make_install_target="install install-doc"
+hostmakedepends="asciidoc"
+short_desc="Transparent bidirectional bridge between Git and Mercurial for Git"
+maintainer="Piraty <piraty1@inbox.ru>"
+license="GPL-2.0-only"
+homepage="https://github.com/mnauw/git-remote-hg"
+distfiles="https://github.com/mnauw/git-remote-hg/archive/refs/tags/v${version}.tar.gz"
+checksum=a6af69ae22f31e86eb060636914088c8421e87f985ba18ae452fe6817aa74057
+python_version=3
+
+post_patch() {
+	vsed -i Makefile \
+		-e 's/prefix := $(HOME)/prefix := $(PREFIX)/'
+}
```

Comment:
Hm, can't you pass `PREFIX=/usr` in `make_build_args`?

Reply:
`PREFIX` isn't used in any `install ...` commands in the makefile

Reply:
sorry, I meant `prefix=/usr` instead.